src/Gui/MainWindow.h

```diff
@@ -29,6 +29,7 @@
 #include <QMdiArea>
 
 #include "Window.h"
+#include "InputHint.h"

```

Comment:
Maybe forward declaration cen be used here? I'm unsure

Reply:
I get a lot of "Incomplete type" errors when I replace it. Maybe there is a way but I feel that it would be a long battle for little to no gain.